config_utilities/test/CMakeLists.txt

```diff
@@ -45,6 +45,7 @@ add_executable(
   tests/string_utils.cpp
   tests/subconfigs.cpp
   tests/traits.cpp
+  tests/update.cpp
```

Comment:
Nice! I'll probably save comments on tests, maybe some of the changes above already solve your questions